lib/block-supports/index.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Block support flags.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Filter the registered blocks to apply the block supports attributes registration.
+ */
+function gutenberg_register_block_supports() {
+	$block_registry         = WP_Block_Type_Registry::get_instance();
+	$registered_block_types = $block_registry->get_all_registered();
+	// Ideally we need a hook to extend the block registration
+	// instead of mutating the block type.
+	foreach ( $registered_block_types as $block_type ) {
+		gutenberg_register_alignment_support( $block_type );
```

Comment:
Should we be doing the same for colors and typography to initialize named and the `style` attribute?  While only `align` seemed to be causing issues with the uses of ServerSideRender component, it would seem to be thorough to register the attributes used in all the hooks similarly.

Reply:
Yes, we should, I just left it for a separate PR.